modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -103,10 +103,6 @@ export default {
   data() {
     return {
       docType: this.moduleName,
-      doc: {
```

Comment:
Now docFields, which is configured for us by the mixin.

Reply:
I prefer docFields because (1) it's analogous to the split field properties, and (2) doc should be a doc, not a thing with a data property and an error property, plus it's not uncommon for the actual existing doc to be a prop in other potential users of the mixin.